robottelo/hosts.py

```diff
@@ -62,7 +62,7 @@ def download_install_rpm(self, repo_url, package_name):
         :raises robottelo.hosts.ContentHostError: If package wasn't installed.
 
         """
-        self.execute(f'wget -nd -r -l1 --no-parent -A \'{package_name}.rpm\' {repo_url}')
+        self.execute(f'curl -O {repo_url}/{package_name}.rpm')
```

Comment:
that is an interesting and welcome change if it works, as curl is installed by default and wget is not. I wonder did @JacobCallahan make a note of that?

Reply:
@swadeley can you make this a blocking request?